commands/FBTextInputCommands.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python
+
+# Copyright (c) 2016, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+import os
+
+import lldb
+import fblldbbase as fb
+import fblldbviewhelpers as viewHelpers
+
+ACCESSIBILITY_ID = 0
+REPLACEMENT_TEXT = 1
+INPUT_TEXT = 0
+
+
+def lldbcommands():
+  return [
+    FBInputTexByAccessibilityIdCommand(),
+    FBInputTexToFirstResponderCommand(),
+  ]
+
+
+class FBInputTexByAccessibilityIdCommand(fb.FBCommand):
+  def name(self):
+    return 'setxt'
```

Comment:
It's just short for text. As per the comment bellow, I based the name on other examples of short hand commands.